OTAcceleratorCore/OTOneToOneCommunicator.m

```diff
@@ -213,6 +223,15 @@ -(void)sessionDidConnect:(OTSession*)session {
     OTError *error;
     [self.session publish:self.publisher error:&error];
     if (error) {
+        [loggingWrapper.logger logEventAction:KLogActionStartPublishing
+                                                     variation:KLogVariationFailure
+                                                    completion:nil];
+        if (self.screenSharingView) {
+            [loggingWrapper.logger logEventAction:KLogActionStartScreensharing
+                                                         variation:KLogVariationFailure
+                                                        completion:nil];
+        }
```

Comment:
Under screen sharing, it will log both. Should we put KLogActionStartPublishing to the else branch?